### PR TITLE
Fix node-streams dep; prep next release

### PR DESCRIPTION
**Pull request by JordanMartinez**
**Description of the change**

Fix `node-streams` dep; prep next release

---

**Checklist:**

- [ ] Added the change to the changelog's "Unreleased" section with a reference to this PR (e.g. "- Made a change (#0000)")
- [ ] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

**Comment by JordanMartinez** [comment]
When I ran the tests locally, the tests just hang after getting to this part:
```
GET /
Headers:
(fromFoldable [(Tuple "content-type" "text/html"),(Tuple "date" "Fri, 29 Apr 2022 14:38:30 GMT"),(Tuple "connection" "close"),(Tuple "transfer-encoding" "chunked")])
Cookies:
Nothing
Response:
<form method='POST' action='/'>
  <input name='text' type='text'>
  <input type='submit'>
</form>
Headers:
(fromFoldable [(Tuple "server" "nginx/1.14.0 (Ubuntu)"),(Tuple "date" "Fri, 29 Apr 2022 14:38:31 GMT"),(Tuple "content-type" "image/svg+xml"),(Tuple "content-length" "967"),(Tuple "connection" "close"),(Tuple "vary" "Accept-Encoding, Accept, Accept-Language"),(Tuple "x-xss-protection" "1; mode=block"),(Tuple "content-security-policy" "default-src 'none'; script-src 'self' *.google-analytics.com; style-src 'self' 'unsafe-inline' fonts.googleapis.com; font-src fonts.gstatic.com; img-src *; connect-src 'self'; manifest-src 'self'; plugin-types 'none'; base-uri 'none'")])
Cookies:
(Just ["_SESSION=KVWq0FxL9kLTb3+QThZrI8Az1clcq0uU7jyahqutgV3SeX0J1qP+262ADKLQTSLFHS1ijFp7aFyB6hB0ELCJw8KcWdC7a/8dz4ZRWoMxXVoiMSp4e9SetaA214XThR/9/zNdRB2iZwz5lVLTu41/3Drz484i75JfY0Ns+rBNz0GpVqgbWbs=; Path=/; Expires=Fri, 29-Apr-2022 16:38:31 GMT; HttpOnly"])
Response:
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
    "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="95" height="20"><linearGradient id="smooth" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1" /><stop offset="1" stop-opacity=".1" /></linearGradient><mask id="round"><rect width="95" height="20" rx="3" fill="#fff" /></mask><g mask="url(#round)"><rect width="49" height="20" fill="#555" /><rect x="49" width="46" height="20" fill="#4c1" /><rect width="95" height="20" fill="url(#smooth)" /></g><g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11"><text x="25.5" y="15" fill="#010101" fill-opacity=".3">pursuit</text><text x="25.5" y="14">pursuit</text><text x="71.0" y="15" fill="#010101" fill-opacity=".3">v7.0.0</text><text x="71.0" y="14">v7.0.0</text></g></svg>Headers:
(fromFoldable [(Tuple "date" "Fri, 29 Apr 2022 14:38:31 GMT"),(Tuple "content-type" "text/html; charset=utf-8"),(Tuple "content-length" "223"),(Tuple "connection" "close"),(Tuple "server" "gunicorn/19.9.0"),(Tuple "location" "/cookies"),(Tuple "access-control-allow-origin" "*"),(Tuple "access-control-allow-credentials" "true")])
Cookies:
(Just ["cookie1=firstcookie; Path=/","cookie2=secondcookie; Path=/"])
Response:
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>Redirecting...</title>
<h1>Redirecting...</h1>
<p>You should be redirected automatically to target URL: <a href="/cookies">/cookies</a>.  If not click the link.
```

**Comment by thomashoneyman** [comment]
See #35 – the hang is OK.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,16 +12,14 @@ Bugfixes:
...
 Breaking changes:
 - Update project and deps to PureScript v0.15.0 (#41 by @JordanMartinez, @sigma-andex)
 
-New features:
-
-Bugfixes:
+## [v7.0.0](https://github.com/purescript-node/purescript-node-http/releases/tag/v7.0.0) - 2022-04-28
 
-Other improvements:
+Due to implementing a breaking change incorrectly, use v10.0.0 instead.
```
Comment (thomashoneyman) [inline CHANGELOG.md]: Use v8.0.0 instead?

Comment (JordanMartinez) [reply CHANGELOG.md]: Ah... That's probably a mistake on a couple of the repos now.

**Comment by JordanMartinez** [comment]
I cancelled the first build since the commit fixing the copy-paste mistake will invalidate it anyway.

**Comment by JordanMartinez** [comment]
Looking at the Actions tab, it seems like the last few builds ran for 6 hours and then failed. I think we really need to fix #35.

**Comment by JordanMartinez** [comment]
I got all but the `testUpgrade` test to work by adding a callback that tears down the server once the last request finishes. However, I couldn't figure out why `testUpgrade`'s server wouldn't close. Some other resource wasn't getting closed, and I believe it was the socket.

**Pull request merged**